src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.scss

```diff
@@ -120,3 +129,15 @@ md-icon {
   color: $delicate;
   font-size: $subhead-font-size-base;
 }
+
+.kd-sparkline {
+  width: 100px;
+  height: 20px;
+  vertical-align: -5px;
```

Comment:
Do you want to center this in a cell? Remove all margins and set line-height: 0. With that you wont have to hardcode anything.